MdeModulePkg/Include/Ppi/DelayedDispatch.h

```diff
@@ -0,0 +1,99 @@
+/**
+
+  PPI that allows a Pei module to request a callback after a minumum
+  delay.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef __EFI_DELAYED_DISPATCH_PPI_H__
+#define __EFI_DELAYED_DISPATCH_PPI_H__

```

Comment:
```suggestion
#ifndef EFI_DELAYED_DISPATCH_PPI_H_
#define EFI_DELAYED_DISPATCH_PPI_H_

```

Reply:
updated to `EFI_DELAYED_DISPATCH_PPI_H_` as requested.